internals/jest/jest.base-config.js

```diff
@@ -1,7 +1,7 @@
 // @ts-check
 // https://jestjs.io/docs/configuration#defaults
 
-/** @type {import('@jest/types').Config.InitialOptions} */
+/** @type {import('ts-jest').JestConfigWithTsJest} */
```

Comment:
why we switch here to ts-jest and keep @jest/type in all three other configs?

Reply:
Ah, I should switch them all for consistency